docs/reference-docs/templates.md

```diff
@@ -0,0 +1,48 @@
+# Templates
+
+A template is a `<filename>.ext.tpl` in a stencil module. 
+
+## Render Behavior
+
+**Location**: By default these write to the same file as the template with `strings.TrimSuffix(name, ".tpl")` in the directory that stencil is being ran in. This can be changed with `file.SetPath`.
+
+**Mode**: By default the mode of the file is taken from template's mode, this can be changed with `file.SetMode`.
+
+## File/Stencil Functions
+
+The `file` and `stencil` functions are scoped to a given template/module (depending on the context) that's currently
+being rendered by stencil.
+
+A complete list of all functions available to a template can be found in pkg.go.dev:
+
+ * [`file`](https://pkg.go.dev/github.com/getoutreach/stencil@v1.5.0/internal/codegen#TplFile) 
```

Comment:
do you have to tie these paths to a version?

Reply:
Nope, we could remove the versions